src/migrations/20230607214325_CreateBlockMetadata.ts

```diff
@@ -0,0 +1,12 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('block_metadata', (table) => {
+    table.integer('blockNumber').primary()
```

Comment:
at 5 seconds per block it will be 337 years until this reaches max int

Reply:
i hope you feel good about that 4 byte savings you're getting on every row

Reply:
you can check my math, but by my calculations, this should save us approximately $1,500,000 over the course of 337 years... great savings!

![2023-06-09-163632_1420x1120_scrot](https://github.com/valora-inc/indexer/assets/569401/d0ba9ab8-0fff-42bc-a074-4b62a3b90b51)

Reply:
under the assumption of course that our "year 2360 problem" costs less than $1,500,000 to fix...